src/apps/backend/app.ts

```diff
@@ -7,10 +7,11 @@ import expressWinston from 'express-winston';
 
 import { AccessTokenServer } from './modules/access-token';
 import { AccountServer } from './modules/account';
+import AvailabilityServer from './modules/availability/rest-api/availability-server';
```

Comment:
We should rename everything `Availability` to `BookingAvailability` (file name, routes, class, typedef etc) to make this better scoped and readable. `Availability` is pretty generic.